### PR TITLE
chore: overhaul readmes

### PR DESCRIPTION
**Pull request by TetieWasTaken**
**Please describe the changes this PR makes and why it should be merged:**
This PR:
- Fixes readmes being out of sync. e.g., some readmes only having a partial contributing section.
```diff
- See [the contribution guide][contributing] if you'd like to submit a PR.
+ Before creating an issue, please ensure that it hasn't already been reported/suggested, and double-check the 
+ [documentation][documentation].
+ See [the contribution guide][contributing] if you'd like to submit a PR.
```
- Adds semantic changes for improved grammar:
```diff
- See also the [Update Guide][guide-update], including updated and removed items in the library.
+ Also see the v13 to v14 [Update Guide][guide-update], which includes updated and removed items from the library.
```
- Adds or changes some about sections to follow the format of other packages.
- Updates the root README*
- Changes documentation links from `https://discord.js.org/#/docs/{package}` to `https://discordjs.dev/docs/packages/{package}`
_*Needs some good checking_

**Status and versioning classification:**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by TetieWasTaken** [comment]
Those are a lot of labels, thank you github actions

File: packages/util/README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@
 	</p>
 </div>
 
+## About
+
+`@discordjs/util` is a collection of utility functions for use with Discord.js.
```
Comment (Jiralite) [inline packages/util/README.md:18]: Do not capitalise discord.js.

**Review by Jiralite — COMMENTED** [review]
I think the update guide should be removed for irrelevant subpackages, to be honest. That probably means removing it from the scripts subpackage.

**Comment by almeidx** [comment]
I think the root README should outline what is present in the monorepo/how to contribute instead of being focused on the discord.js package itself

**Comment by Jiralite** [comment]
> I think the root README should outline what is present in the monorepo/how to contribute instead of being focused on the discord.js package itself

Totally agree.

**Comment by TetieWasTaken** [comment]
> I think the update guide should be removed for irrelevant subpackages, to be honest. That probably means removing it from the scripts subpackage.

I partially agree here, I do think it makes sense. But I also think the links section should be global and the same everywhere. Since it's meant as a 'guide' to help you around.

File: README.md [hunk 2]
```diff
@@ -18,23 +18,25 @@
...
+- [`@discordjs/brokers`] ([source][brokers-source]) - A collection of brokers for use with discord.js
+- [`@discordjs/builders`] ([source][builders-source]) - A utility package for easily building Discord API payloads
+- [`@discordjs/collection`] ([source][collection-source]) - A powerful utility data structure
+- [`@discordjs/core`] ([source][core-source]) - A thinly abstracted wrapper around the core components of the Discord API
+- [`@discordjs/formatters`] ([source][formatters-source]) - A collection of functions for formatting strings
+- [`@discordjs/proxy`] ([source][proxy-source]) - A wrapper around `@discordjs/rest` for running an HTTP proxy
+- [`@discordjs/rest`] ([source][rest-source]) - A module for interacting with the Discord REST API
+- [`@discordjs/voice`] ([source][voice-source]) - A module for interacting with the Discord Voice API
+- [`@discordjs/util`] ([source][util-source]) - A collection of utility functions
+- [`@discordjs/ws`] ([source][ws-source]) - A wrapper around Discord's gateway
```
Comment (Jiralite) [inline README.md:39]: Why the []?

File: README.md [hunk 3]
```diff
@@ -143,3 +85,15 @@ nudge in the right direction, please don't hesitate to join our official [discor
...
+[builders-source]: https://github.com/discordjs/discord.js/tree/main/packages/builders
+[collection-source]: https://github.com/discordjs/discord.js/tree/main/packages/collection
+[core-source]: https://github.com/discordjs/discord.js/tree/main/packages/core
+[formatters-source]: https://github.com/discordjs/discord.js/tree/main/packages/formatters
+[proxy-source]: https://github.com/discordjs/discord.js/tree/main/packages/proxy
+[rest-source]: https://github.com/discordjs/discord.js/tree/main/packages/rest
+[voice-source]: https://github.com/discordjs/discord.js/tree/main/packages/voice
+[util-source]: https://github.com/discordjs/discord.js/tree/main/packages/util
+[ws-source]: https://github.com/discordjs/discord.js/tree/main/packages/ws
+[good-first-issue]: https://github.com/discordjs/discord.js/labels/good%20first%20issue
```
Comment (Jiralite) [inline README.md:99]:
```suggestion
[good-first-issue]: https://github.com/discordjs/discord.js/contribute
```

**Review by Jiralite — CHANGES_REQUESTED** [review]

File: packages/scripts/README.md [hunk 4]
```diff
@@ -18,14 +18,16 @@
...
+  Also see the v13 to v14 [Update Guide][guide-update], which includes updated and removed items from the library.
 - [discord.js Discord server][discord]
 - [Discord API Discord server][discord-api]
 - [GitHub][source]
 - [Related libraries][related-libs]
 
 ## Contributing
 
+Before creating an issue, please ensure that it hasn't already been reported/suggested, and double-check the
+[documentation][documentation].
```
Comment (Jiralite) [inline packages/scripts/README.md:30]: This is on 2 lines? Make it 1.

Also, for all subpackages, change the link to `https://discordjs.dev/docs/packages/{package}`. For apps/, make it https://discordjs.dev.

Comment (TetieWasTaken) [reply packages/scripts/README.md:30]: Some packages don't have a `https://discordjs.dev/` page, what should we do with them?

File: README.md [hunk 5]
```diff
@@ -120,18 +64,15 @@ client.login(TOKEN);
 
 ## Contributing
 
-Before creating an issue, please ensure that it hasn't already been reported/suggested, and double-check the
-[documentation][documentation].  
-See [the contribution guide][contributing] if you'd like to submit a PR.
+Please read through our [contributing guidelines][contributing] before making a PR. We welcome contributions of all kinds, not just code! If you're stuck for ideas, look at the [good first issue][good-first-issue] label on issues in the repository. If you have any questions about the project, feel free to ask them in [discord.js' Discord server][discord]. Before creating your own issue or pull request, always check to see if one already exists! Also, don't rush into creating a pull request; it's always better to take your time and ensure you're doing it correctly.
```
Comment (almostSouji) [inline README.md:67]:
```suggestion
Please read through our [contribution guidelines][contributing] before starting a pull request. We welcome contributions of all kinds, not just code! If you're stuck for ideas, look for the [good first issue][good-first-issue] label on issues in the repository. If you have any questions about the project, feel free to ask them on [Discord][discord]. Before creating your own issue or pull request, always check to see if one already exists! Don't rush contributions, take your time and ensure you're doing it correctly.
```
wording and flow

File: README.md [hunk 6]
```diff
@@ -120,18 +64,15 @@ client.login(TOKEN);
...
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
```
Comment (almostSouji) [inline README.md:71]:
```suggestion
If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please join our [Discord server][discord].
```

clarity, flow

**Review by almostSouji — CHANGES_REQUESTED** [review]
I agree that there is no need for all packages to have the same or even necessarily a similar readme. the readme should be highly relevant to each package.

linking the updating guides makes no sense to do in any package save the assembled API wrapper, for example

File: README.md [hunk 7]
```diff
@@ -18,23 +18,25 @@
...
+This repository is a monorepo that contains multiple packages. The main and most popular package is [`discord.js`][source], which is the one you are most likely looking for. All packages have seperate [release][github-releases] cycles and version numbers.
 
-- Object-oriented
-- Predictable abstractions
-- Performant
-- 100% coverage of the Discord API
+## Getting started
 
-## Installation
+To get started with a package, navigate to its directory (packages/{packagename}) and follow the instructions in its README file. You can also find links to the documentation, guide, and other resources in the [Links](#links) section.
```
[the remark this answers is not in the document]
Comment (TetieWasTaken) [reply README.md:25]: I was debating this as well. But I also wanted to add a paragraph like such that helps people navigate the monorepo.

[the remark this answers is not in the document]
Comment (Jiralite) [reply README.md:25]: Updating this and the section below seems like a chore. Also, the optional packages should be moved to the respective packages. They are not for every package.

File: packages/scripts/README.md (hunk 4 above)
Comment (Jiralite) [reply packages/scripts/README.md]: Don't include them. You said these documents are "out of sync", but who said they should have been in sync in the first place?

> the readme should be highly relevant to each package.

Comment (TetieWasTaken) [reply packages/scripts/README.md]: I meant the inconsistency in about (as in some don't have them, some do), contributing and the help section. I do not mean with "in sync" that they have to be exactly the same in their description, examples, etc..

File: README.md [hunk 8]
```diff
@@ -120,18 +52,15 @@ client.login(TOKEN);
...
+Please read through our [contribution guidelines][contributing] before starting a pull request. We welcome contributions of all kinds, not just code! If you're stuck for ideas, look for the [good first issue][good-first-issue] label on issues in the repository. If you have any questions about the project, feel free to ask them on [Discord][discord]. Before creating your own issue or pull request, always check to see if one already exists! Don't rush contributions, take your time and ensure you're doing it correctly.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please join our [Discord server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline README.md:61]:
```suggestion
[website]: https://discord.js.org
```

File: README.md [hunk 9]
```diff
@@ -120,18 +52,15 @@ client.login(TOKEN);
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs
+[documentation]: https://discordjs.dev/docs/packages
```
Comment (iCrawl) [inline README.md:63]:
```suggestion
[documentation]: https://discord.js.org/docs
```

File: apps/guide/README.md [hunk 10]
```diff
@@ -13,27 +13,32 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline apps/guide/README.md:41]:
```suggestion
[website]: https://discord.js.org
```

File: apps/guide/README.md [hunk 11]
```diff
@@ -13,27 +13,32 @@
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
 [documentation]: https://discord.js.org/
```
Comment (iCrawl) [inline apps/guide/README.md:43]:
```suggestion
[documentation]: https://discord.js.org/docs
```

File: apps/website/README.md [hunk 12]
```diff
@@ -18,22 +18,23 @@
...
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
 [documentation]: https://discord.js.org/
```
Comment (iCrawl) [inline apps/website/README.md:39]:
```suggestion
[documentation]: https://discord.js.org/docs
```

File: apps/website/README.md [hunk 13]
```diff
@@ -18,22 +18,23 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline apps/website/README.md:37]:
```suggestion
[website]: https://discord.js.org
```

File: packages/actions/README.md [hunk 14]
```diff
@@ -16,26 +16,25 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/actions/README.md:36]:
```suggestion
[website]: https://discord.js.org
```

File: packages/api-extractor-utils/README.md [hunk 15]
```diff
@@ -16,26 +16,25 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/api-extractor-utils/README.md:36]:
```suggestion
[website]: https://discord.js.org
```

File: packages/brokers/README.md [hunk 16]
```diff
@@ -103,12 +103,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/brokers
+[documentation]: https://discordjs.dev/docs/packages/brokers
```
Comment (iCrawl) [inline packages/brokers/README.md:110]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/brokers/stable
```

File: packages/brokers/README.md [hunk 17]
```diff
@@ -103,12 +103,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/brokers/README.md:108]:
```suggestion
[website]: https://discord.js.org
```

File: packages/builders/README.md [hunk 18]
```diff
@@ -52,13 +54,12 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
 [example]: https://github.com/discordjs/discord.js/blob/main/packages/builders/docs/examples/Slash%20Command%20Builders.md
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/builders/README.md:60]:
```suggestion
[website]: https://discord.js.org
```

File: packages/builders/README.md [hunk 19]
```diff
@@ -52,13 +54,12 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/builders
+[documentation]: https://discordjs.dev/docs/packages/builders
```
Comment (iCrawl) [inline packages/builders/README.md:62]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/builders/stable
```

File: packages/collection/README.md [hunk 20]
```diff
@@ -50,12 +50,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/collection/README.md:55]:
```suggestion
[website]: https://discord.js.org
```

File: packages/collection/README.md [hunk 21]
```diff
@@ -50,12 +50,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/collection
+[documentation]: https://discordjs.dev/docs/packages/collection
```
Comment (iCrawl) [inline packages/collection/README.md:57]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/collection/stable
```

File: packages/core/README.md [hunk 22]
```diff
@@ -98,12 +98,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/core/README.md:103]:
```suggestion
[website]: https://discord.js.org
```

File: packages/core/README.md [hunk 23]
```diff
@@ -98,12 +98,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/
+[documentation]: https://discordjs.dev/docs/packages/core
```
Comment (iCrawl) [inline packages/core/README.md:105]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/core/stable
```

File: packages/discord.js/README.md [hunk 24]
```diff
@@ -126,12 +126,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/discord.js/README.md:131]:
```suggestion
[website]: https://discord.js.org
```

File: packages/discord.js/README.md [hunk 25]
```diff
@@ -126,12 +126,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs
+[documentation]: https://discordjs.dev/docs/packages/discord.js
```
Comment (iCrawl) [inline packages/discord.js/README.md:133]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/discord.js/stable
```

File: packages/docgen/README.md [hunk 26]
```diff
@@ -27,14 +27,15 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/docgen/README.md:38]:
```suggestion
[website]: https://discord.js.org
```

File: packages/docgen/README.md [hunk 27]
```diff
@@ -27,14 +27,15 @@
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
 [documentation]: https://discord.js.org/
```
Comment (iCrawl) [inline packages/docgen/README.md:40]:
```suggestion
```

File: packages/formatters/README.md [hunk 28]
```diff
@@ -66,9 +66,9 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/
+[documentation]: https://discordjs.dev/docs/packages/formatters
```
Comment (iCrawl) [inline packages/formatters/README.md:71]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/formatters/stable
```

File: packages/formatters/README.md [hunk 29]
```diff
@@ -66,9 +66,9 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/formatters/README.md:69]:
```suggestion
[website]: https://discord.js.org
```

File: packages/next/README.md [hunk 30]
```diff
@@ -43,12 +42,10 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/next/README.md:47]:
```suggestion
[website]: https://discord.js.org
```

File: packages/next/README.md [hunk 31]
```diff
@@ -43,12 +42,10 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/next
```
Comment (iCrawl) [inline packages/next/README.md:51]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/next/stable
```

File: packages/proxy-container/README.md [hunk 32]
```diff
@@ -53,22 +53,23 @@ const rest = new REST({
...
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
 [documentation]: https://discord.js.org/
```
Comment (iCrawl) [inline packages/proxy-container/README.md:74]:
```suggestion
```

File: packages/proxy-container/README.md [hunk 33]
```diff
@@ -53,22 +53,23 @@ const rest = new REST({
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/proxy-container/README.md:72]:
```suggestion
[website]: https://discord.js.org
```

File: packages/proxy/README.md [hunk 34]
```diff
@@ -50,12 +50,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/proxy
+[documentation]: https://discordjs.dev/docs/packages/proxy
```
Comment (iCrawl) [inline packages/proxy/README.md:57]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/proxy/stable
```

File: packages/proxy/README.md [hunk 35]
```diff
@@ -50,12 +50,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/proxy/README.md:55]:
```suggestion
[website]: https://discord.js.org
```

File: packages/rest/README.md [hunk 36]
```diff
@@ -95,12 +99,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/rest/README.md:104]:
```suggestion
[website]: https://discord.js.org
```

File: packages/rest/README.md [hunk 37]
```diff
@@ -95,12 +99,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/rest
+[documentation]: https://discordjs.dev/docs/packages/rest
```
Comment (iCrawl) [inline packages/rest/README.md:106]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/rest/stable
```

File: packages/scripts/README.md [hunk 38]
```diff
@@ -16,26 +16,24 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/scripts/README.md:35]:
```suggestion
[website]: https://discord.js.org
```

File: packages/ui/README.md [hunk 39]
```diff
@@ -18,22 +18,23 @@
...
 See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/ui/README.md:37]:
```suggestion
[website]: https://discord.js.org
```

File: packages/ui/README.md [hunk 40]
```diff
@@ -18,22 +18,23 @@
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
 [documentation]: https://discord.js.org/
```
Comment (iCrawl) [inline packages/ui/README.md:39]:
```suggestion
```

File: packages/util/README.md [hunk 41]
```diff
@@ -43,12 +47,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/util/README.md:52]:
```suggestion
[website]: https://discord.js.org
```

File: packages/util/README.md [hunk 42]
```diff
@@ -43,12 +47,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/util
+[documentation]: https://discordjs.dev/docs/packages/util
```
Comment (iCrawl) [inline packages/util/README.md:54]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/util/stable
```

File: packages/voice/README.md [hunk 43]
```diff
@@ -88,12 +88,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/voice/README.md:93]:
```suggestion
[website]: https://discord.js.org
```

File: packages/voice/README.md [hunk 44]
```diff
@@ -88,12 +88,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/voice
+[documentation]: https://discordjs.dev/docs/packages/voice
```
Comment (iCrawl) [inline packages/voice/README.md:95]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/voice/stable
```

File: packages/ws/README.md [hunk 45]
```diff
@@ -179,12 +179,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
...
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
 [website-source]: https://github.com/discordjs/discord.js/tree/main/apps/website
-[documentation]: https://discord.js.org/#/docs/ws
+[documentation]: https://discordjs.dev/docs/packages/ws
```
Comment (iCrawl) [inline packages/ws/README.md:186]:
```suggestion
[documentation]: https://discord.js.org/docs/packages/ws/stable
```

File: packages/ws/README.md [hunk 46]
```diff
@@ -179,12 +179,11 @@ See [the contribution guide][contributing] if you'd like to submit a PR.
 
 ## Help
 
-If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle
-nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
+If you don't understand something in the documentation, you are experiencing problems, or you just need a gentle nudge in the right direction, please don't hesitate to join our official [discord.js Server][discord].
 
-[website]: https://discord.js.org/
+[website]: https://discordjs.dev/
```
Comment (iCrawl) [inline packages/ws/README.md:184]:
```suggestion
[website]: https://discord.js.org
```

**Review by iCrawl — CHANGES_REQUESTED** [review]

**Comment by TetieWasTaken** [comment]
Might I ask why the revert of `discord.js.org` -> `discordjs.dev`? I first got told to change them and now we're reverting it back—I'm not against the decision but such disagreements can be confusing.

**Comment by iCrawl** [comment]
Yeah its honestly not your fault, no one knew when I was about to switch the "main" domain to the new site, which is what I initiated now 👍

**Pull request merged**